File: desktop/src/main/java/com/youssef12347/exchange/api/model/Graph.java
package com.youssef12347.exchange.api.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Graph {
    @SerializedName("date")
    public List date;
    @SerializedName("mean_rate_lbp_to_usd")
    public List mean_rate_lbp_to_usd;
    @SerializedName("mean_rate_usd_to_lbp")
    public List mean_rate_usd_to_lbp;
}

File: android/app/src/main/java/com/kmz07/currencyexchange/api/model/Graph.kt
package com.kmz07.currencyexchange.api.model

import com.google.gson.annotations.SerializedName

class Graph {
    @SerializedName("date")
    var graph_dates: ArrayList<String>? = null
    @SerializedName("mean_rate_lbp_to_usd")
    var mean_rate_lbp_to_usd: ArrayList<Float>? = null
    @SerializedName("mean_rate_usd_to_lbp")
    var mean_rate_usd_to_lbp: ArrayList<Float>? = null
}